assets/js/gutenberg-plugin.js

```diff
@@ -131,14 +132,136 @@ const RenderDistributedTo = () => {
  * Render the distributed from component
  */
 const RenderDistributedFrom = () => {
+	const [ isOpen, setOpen ] = useState( false );
+	const openModal = () => setOpen( true );
+	const closeModal = () => setOpen( false );
+
+	if ( ! parseInt( dtGutenberg.unlinked ) ) {
+		return (
+			<div>
+				<span id="distributed-from">
+					{ sprintf(
+						/* translators: 1: Syndication date and time. */
+						__(
+							'Pulled & linked on %1$s from %2$s',
+							'distributor'
+						),
+						dtGutenberg.syndicationTime,
+						dtGutenberg.originalLocationName
+					) }
+				</span>
+				<span id="distributed-data">
+					{ __( 'Updating the ', 'distributor' ) }
+					<a href={ dtGutenberg.postUrl }>
+						{ __( 'Original Content', 'distributor' ) },
+					</a>
```

Comment:
```suggestion
						{ __( 'Original Content', 'distributor' ) }
					</a>
					{ ', ' }
```

Adding extra space here will fix the spacing issue.
![image](https://github.com/user-attachments/assets/ac49f449-6018-4ba4-b276-b8ceb5837583)